src/main/resources/messages_nl_NL.properties

```diff
@@ -29,51 +29,51 @@ pageNum=Paginanummer
 sizes.small=Klein
 sizes.medium=Medium
 sizes.large=Groot
-sizes.x-large=Extra Groot
-error.pdfPassword=Het PDF document is beveiligd met een wachtwoord en het wachtwoord is niet ingevoerd of was onjuist
+sizes.x-large=Extra groot
+error.pdfPassword=Het PDF document is beveiligd met een wachtwoord en het wachtwoord is niet ingevoerd of is onjuist
 delete=Verwijderen
 username=Gebruikersnaam
 password=Wachtwoord
 welcome=Welkom
-property=Property
-black=Black
-white=White
-red=Red
-green=Green
-blue=Blue
-custom=Custom...
-WorkInProgess=Work in progress, May not work or be buggy, Please report any ploblems!
-poweredBy=Powered by
-yes=Yes
-no=No
-changedCredsMessage=Credentials changed!
-notAuthenticatedMessage=User not authenticated.
-userNotFoundMessage=User not found.
-incorrectPasswordMessage=Current password is incorrect.
-usernameExistsMessage=New Username already exists.
+property=Eigenschap
+black=Zwart
+white=Wit
+red=Rood
+green=Groen
+blue=Blauw
+custom=Aangepast...
+WorkInProgess=Werk in uitvoering. Werkt mogelijk niet of bevat fouten. Meld eventuele problemen!
+poweredBy=Mogelijk gemaakt door
+yes=Ja
+no=Nee
+changedCredsMessage=Inloggegevens gewijzigd!
+notAuthenticatedMessage=Gebruiker niet ingelogd.
+userNotFoundMessage=Gebruiker niet gevonden.
+incorrectPasswordMessage=Huidige wachtwoord is onjuist.
+usernameExistsMessage=Nieuwe gebruikersnaam bestaat al.
 
 
 ###############
 #   Pipeline  #
 ###############
-pipeline.header=Pipeline Menu (Alpha)
-pipeline.uploadButton=Upload Custom
-pipeline.configureButton=Configure
-pipeline.defaultOption=Custom
-pipeline.submitButton=Submit
+pipeline.header=Pijplijn menu (Alpha)
+pipeline.uploadButton=Aangepast uploaden
+pipeline.configureButton=Configureren
+pipeline.defaultOption=Aangepast
+pipeline.submitButton=Opslaan
 
 ######################
 #  Pipeline Options  #
 ######################
-pipelineOptions.header=Pipeline Configuration
-pipelineOptions.pipelineNameLabel=Pipeline Name
-pipelineOptions.saveSettings=Save Operation Settings
-pipelineOptions.pipelineNamePrompt=Enter pipeline name here
-pipelineOptions.selectOperation=Select Operation
-pipelineOptions.addOperationButton=Add operation
-pipelineOptions.pipelineHeader=Pipeline:
-pipelineOptions.saveButton=Download
-pipelineOptions.validateButton=Validate
+pipelineOptions.header=Pijplijn onfiguratie
```

Comment:
Pijplijn **_c_**onfiguratie
(c mist)

Reply:
Do you want to raise a new PR to resolve this?

Reply:
If Michel is not faster, sure, I just saw the release notes and peaked quickly ;)